app/Http/Controllers/PollController.php

```diff
@@ -32,6 +33,27 @@ public function create(Request $request): JsonResponse
         }
 
         $poll->save();
+
         return response()->json($poll, status: 201);
     }
+
+    public function getPoll(Request $request, int $id): JsonResponse
+    {
+        $poll = Poll::find($id);
+        $votes = $poll->votes()->get();
+        $upvotes = 0;
+        $downvotes = 0;
+        foreach ($votes as $value) {
+            if ($value->description == 0) {
+                $downvotes += 1;
+            }
+
+            if ($value->description == 1) {
+                $upvotes += 1;
+            }
+            unset($value); 
+        }
+
+        return response()->json([$poll, $upvotes, $downvotes], status: 200);
```

Comment:
thats not a descriptive json response. I would like to have a json response like this, because its a single object that i'm requesting.

```json
{
    "id": 1,
    "name": "Queen left off, quite out of his great wig.' The judge, by the prisoner to--to somebody.' 'It must be kind to them,' thought Alice, 'or perhaps they won't walk the way out of his great wig.' The.",
    "description": "For instance, suppose it doesn't matter a bit,' said the March Hare. 'He denies it,' said Alice, a little bottle that stood near. The three soldiers wandered about in the window, she suddenly spread.",
    "endDate": "2023-03-07",
    "phase": 1,
   "upvotes": 81,
    "downvotes": 67,
    "created_at": "2023-01-31T21:41:16.000000Z",
    "updated_at": "2023-01-31T21:41:16.000000Z"
},
```

but your json response is like this

```json
[

    {
        "id": 1,
        "name": "peterabschaffen",
        "description": "bitte einfach weghauen",
        "end_date": "2023-02-26",
        "phase": 1,
        "created_at": null,
        "updated_at": null
    },
    0,
    0

]
```

as api user, i wouldnt know why there are some 0s in the document.